documentation/assembly.md

```diff
@@ -1,19 +1,23 @@
 # Assembly guide
 
+The bill of materials for the assembly can be found [here](../source/BoM/bom.md)
 
-![Wiring diagram](/documentation/images/wiring-diagram.png)
+![Wiring diagram](https://docs.google.com/drawings/d/e/2PACX-1vTsVebLCV5xOCOP7e8MOFAZS-pnqpgaU0UeKEsrxiiekd_nlwJRW8BvfpLmfiG9eoDodftNeQdaIt7_/pub?w=960&h=720)
 
 1. Mount the relay in the bottom half of the case
 2. Prepare 6 strands of the unsulated cable, 2 blue, 2 brown, 1 yellow/green
 3. Crimp fast release terminals to both ends of the  yellow/green wires
 4. Crimp fast release terminals to one end of each of the brown and blue wires
-5. Screw the bare wire terminals into the NC and COM terminals of the relay according to the diagram
+5. Crimp blue blade terminals to the other end of each brown and blue wire
+6. Screw the blade terminals into the relay screw terminals
 7. Attach the fast release terminals to the C13 and C14 connectors
-9. Place the C13/14 connectors in the right size holes that are cut into the case
-10. Place the top half of the case on top of the bottom half
+8. Place the C13/14 connectors in the right size holes that are cut into the case
+9.  Place the top half of the case on top of the bottom half
 
 ![Example assembly](/documentation/images/example-assembly.jpg)
 
-> **Note:** The above image didn't use the correct colors as electrical wiring standards vary depending on the country. Thus, it should only be used as a reference. Further builds must use the colors indicated in the textual description above.
-
 > **Note:** If the relay board feels flimsy inside the case you may use M2.5x6mm screws to hold it in place
+
+> **Note:** Ensure that the correct colours are used for line (brown), earth (yellow/green) and neutral (blue)
```

Comment:
'Ensure that the correct colours are used for line (brown), earth (yellow/green) and neutral (blue) following EU/UK IEC convention'